[2023-05-18T10:40:51.961+0530] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: combined_dag.consume_from_kafka_task manual__2023-05-18T05:08:34.904884+00:00 [queued]>
[2023-05-18T10:40:51.972+0530] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: combined_dag.consume_from_kafka_task manual__2023-05-18T05:08:34.904884+00:00 [queued]>
[2023-05-18T10:40:51.972+0530] {taskinstance.py:1331} INFO - Starting attempt 1 of 4
[2023-05-18T10:40:51.989+0530] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): consume_from_kafka_task> on 2023-05-18 05:08:34.904884+00:00
[2023-05-18T10:40:51.994+0530] {standard_task_runner.py:57} INFO - Started process 7938 to run task
[2023-05-18T10:40:51.998+0530] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'combined_dag', 'consume_from_kafka_task', 'manual__2023-05-18T05:08:34.904884+00:00', '--job-id', '398', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmp4xpm9zoy']
[2023-05-18T10:40:51.999+0530] {standard_task_runner.py:85} INFO - Job 398: Subtask consume_from_kafka_task
[2023-05-18T10:40:52.064+0530] {task_command.py:410} INFO - Running <TaskInstance: combined_dag.consume_from_kafka_task manual__2023-05-18T05:08:34.904884+00:00 [running]> on host nizam-HP-15-Notebook-PC
[2023-05-18T10:40:52.182+0530] {taskinstance.py:1568} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='combined_dag' AIRFLOW_CTX_TASK_ID='consume_from_kafka_task' AIRFLOW_CTX_EXECUTION_DATE='2023-05-18T05:08:34.904884+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-05-18T05:08:34.904884+00:00'
[2023-05-18T10:40:52.186+0530] {conn.py:380} INFO - <BrokerConnection node_id=bootstrap-0 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: connecting to localhost:9092 [('127.0.0.1', 9092) IPv4]
[2023-05-18T10:40:52.186+0530] {conn.py:1205} INFO - Probing node bootstrap-0 broker version
[2023-05-18T10:40:52.187+0530] {conn.py:410} INFO - <BrokerConnection node_id=bootstrap-0 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: Connection complete.
[2023-05-18T10:40:52.303+0530] {conn.py:1267} INFO - Broker version identified as 2.5.0
[2023-05-18T10:40:52.304+0530] {conn.py:1268} INFO - Set configuration api_version=(2, 5, 0) to skip auto check_version requests on startup
[2023-05-18T10:40:52.307+0530] {subscription_state.py:171} INFO - Updating subscribed topics to: ('test',)
[2023-05-18T10:40:52.337+0530] {cluster.py:371} INFO - Group coordinator for group1 is BrokerMetadata(nodeId='coordinator-1001', host='localhost', port=9092, rack=None)
[2023-05-18T10:40:52.338+0530] {base.py:693} INFO - Discovered coordinator coordinator-1001 for group group1
[2023-05-18T10:40:52.339+0530] {base.py:741} INFO - Starting new heartbeat thread
[2023-05-18T10:40:52.346+0530] {consumer.py:348} INFO - Revoking previously assigned partitions set() for group group1
[2023-05-18T10:40:52.348+0530] {conn.py:380} INFO - <BrokerConnection node_id=coordinator-1001 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: connecting to localhost:9092 [('127.0.0.1', 9092) IPv4]
[2023-05-18T10:40:52.349+0530] {conn.py:410} INFO - <BrokerConnection node_id=coordinator-1001 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: Connection complete.
[2023-05-18T10:40:52.350+0530] {conn.py:919} INFO - <BrokerConnection node_id=bootstrap-0 host=localhost:9092 <connected> [IPv4 ('127.0.0.1', 9092)]>: Closing connection. 
[2023-05-18T10:40:52.451+0530] {base.py:450} INFO - (Re-)joining group group1
[2023-05-18T10:40:55.206+0530] {base.py:335} INFO - Successfully joined group group1 with generation 240
[2023-05-18T10:40:55.206+0530] {subscription_state.py:257} INFO - Updated partition assignment: []
[2023-05-18T10:40:55.206+0530] {consumer.py:245} INFO - Setting newly assigned partitions set() for group group1
[2023-05-18T10:44:37.094+0530] {process_utils.py:131} INFO - Sending Signals.SIGTERM to group 7938. PIDs of all processes in the group: [7938]
[2023-05-18T10:44:37.095+0530] {process_utils.py:86} INFO - Sending the signal Signals.SIGTERM to group 7938
[2023-05-18T10:44:37.095+0530] {taskinstance.py:1540} ERROR - Received SIGTERM. Terminating subprocesses.
[2023-05-18T10:44:37.096+0530] {logging_mixin.py:149} INFO - Error occurred in consume_from_kafka: Task received SIGTERM signal
[2023-05-18T10:44:37.097+0530] {base.py:748} INFO - Stopping heartbeat thread
[2023-05-18T10:44:37.097+0530] {python.py:183} INFO - Done. Returned value was: None
[2023-05-18T10:44:37.122+0530] {taskinstance.py:1368} INFO - Marking task as SUCCESS. dag_id=combined_dag, task_id=consume_from_kafka_task, execution_date=20230518T050834, start_date=20230518T051051, end_date=20230518T051437
[2023-05-18T10:44:37.147+0530] {process_utils.py:79} INFO - Process psutil.Process(pid=7938, status='terminated', exitcode=0, started='10:40:51') (7938) terminated with exit code 0
